### ISSUE TITLE
IN PRODUCTION: import express from 'express';  SyntaxError: Unexpected reserved word

### ISSUE THREAD

**Issue by racci** [angular-fullstack/generator-angular-fullstack#1831]
Hi, 
In production
Node | v0.12.5
npm | 2.11.2

In local, 
Node | v0.12.5
npm | 3.7.5
 generator-angular-fullstack "version": "2.1.1"

In localhost,
Everything works fine in project's main directory, there is not any problem.
"grunt build" and "grunt shipit:production deploy"  also work without any error or warning. 

However, when I run project under dist directory, after "grunt build", it gives following error
Also in production I try to start project with pm2 start or "node app.js" it gives same following error.

at Function.Module._load (module.js:310:12)
at Function.<anonymous> (/home/deployer/.global_npm_packages/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
at Object.<anonymous> (/home/deployer/.global_npm_packages/lib/node_modules/pm2/lib/ProcessContainerFork.js:46:21)
at Module._compile (module.js:460:26)
at Object.Module._extensions..js (module.js:478:10)
at Module.load (module.js:355:32)
/var/www/xxx/releases/20160426184925/server/app.js:7
 import express from 'express';
 ^^^^^^
SyntaxError: Unexpected reserved word
at exports.runInThisContext (vm.js:73:16)
at Module._compile (module.js:443:25)
at Object.Module._extensions..js (module.js:478:10)
at Module.load (module.js:355:32)
at Function.Module._load (module.js:310:12)
at Function.<anonymous> (/home/deployer/.global_npm_packages/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
at Object.<anonymous> (/home/deployer/.global_npm_packages/lib/node_modules/pm2/lib/ProcessContainerFork.js:46:21)
at Module._compile (module.js:460:26)
at Object.Module._extensions..js (module.js:478:10)
at Module.load (module.js:355:32)

**Comment by Koslun** [comment]
The generator is currently at version 3.6.1, or almost one year after the 2.1.1 version. If this is a new project I would advise you to use the newer version.

If it's an older project using an older version of the generator I think you should probably give a little more information about the choices you made. Might also want to check for fixes between the current version and your version.

Looking at the error I'm thinking it's an error relating to es6 support, specifically related to `import`. Babel supports it for sure. Node 0.12 for sure does not. Since it supposedly works in localhost I'd guess it's Babel not translating the scripts from es6 to es5.

So would either fix the Babel pipeline or change all the `import `statements to regular `var `statements. I.e. `var express = require('express');` instead of `import express from 'express'`. Might be a whole lot of es6 syntax that needs to be fixed then. Think generating a new project and taking in a lot of the files from the new version might even be faster depending on your project.

**Comment by racci** [comment]
Thank you for your answer. 
Yes you right, the best way is to use the latest stable versions. I used Node v0.12 for my previous couple projects with previous generator-angular-fullstack versions. For this time I used es6 supported version.

Actually,  I handled it deleting local dist folder inside and production folder /var/www/xxxx/current and /var/www/xxxx/releases. Then deployed with shipit. It works right now.

As you said, best way to use latest stable compatible versions together.
